### PR TITLE
Make all <a href="javascript:;"> form submit buttons accessible

### PR DESCRIPTION
**Pull request by PaulMorris**
Working towards accessible submit buttons on all forms, this PR converts the submit buttons that were `<a href="javascript:;">` to accessible `<button type="submit">` buttons.  It includes some preliminary CSS rules for the new buttons that will likely revisit after the present accessibility work has finished.

Tested by manually checking a few of the pages with the HTML_CodeSniffer bookmarklet to make sure the accessibility errors for missing submit buttons was gone.  I also checked the formatting for the template that involved moving a form outside of a div to get the buttons inside the form (user-account-edit-system-admin.jsp).  Finally, web UI tests pass successfully.

Issue #553 Use accessible submit buttons on all forms

### CODE REVIEW

File: psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp [hunk 1]
```diff
@@ -128,11 +128,7 @@
                     <a href="<c:url value="/forgotpassword" />">Forgot Password?</a>
                   </div>
                   <div class="buttons">
-                    <a href="javascript:;" id="btnLogin" class="purpleBtn">
-                      <span class="btR">
-                        <span class="btM">Login</span>
-                      </span>
-                    </a>
+                    <button id="btnLogin" class="purpleBtn" type="submit">Login</button>
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/error.jsp]: Thanks for remembering to add the class back!

File: psm-app/cms-web/WebContent/css/style.css [hunk 2]
```diff
@@ -786,7 +786,8 @@ input.date{
   float:left;
   margin-left: 5%;
 }
-.loginPanel  a{
+.loginPanel  a,
+#btnLogin {
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/css/style.css]: I like that you've managed to preserve the styling on the new button.  Thanks!

Comment (PaulMorris) [reply psm-app/cms-web/WebContent/css/style.css]: Thanks!  I'm looking forward to improving the styling on the new buttons at some point.

**Comment by jasonaowen** [comment]
Sorry I missed this earlier! It seems that the links this PR removes were only working by the grace of JavaScript, and it's not entirely clear to me how switching them to buttons works. Some of them, like the `btnLogin`, are simple and correct, and we should remove the associated JavaScript from `script.js`. Some of them are more complicated, lke `filterBtn`, for example: it uses JavaScript to update the DOM, but doesn't seem to `POST` anything anywhere, which it looks like maybe now it does?

**Comment by PaulMorris** [comment]
Thanks @jasonaowen !  I'll look into these and do a follow-up PR.

**Pull request merged**